Improve documentation about a key property (#576)
snabbdom/snabbdom: README.md

an object, where `.key` is the key and the value is the
[`.elm`](#elm--element) property created.

> The `.key` property must be unique among sibling elements.

Reviewer: ```suggestion
If provided, the `.key` property must be unique among sibling elements.
```

> `.key` property should be unique for sibling elements [(here is how key map is created in during the patch phase)](https://github.com/snabbdom/snabbdom/blob/0fe8fd323770012b224253b8857edaa37f77dbe2/src/snabbdom.ts#L248)

Reviewer: ```suggestion
The `.key` property must be unique among sibling elements.
```
